greenboot.spec

```diff
@@ -169,6 +169,10 @@ install -DpZm 0755 usr/lib/greenboot/check/wanted.d/* %{buildroot}%{_prefix}/lib
 %{_prefix}/lib/%{name}/check/required.d/02_watchdog.sh
 
 %changelog
+* Thu Jul 15 2022 Sayan Paul <paul.sayan@gmail.com> - 0.15.0-1
```

Comment:
This date is not correct, it's either Fri 15, or Thu 14, but this should now become today's date.

Reply:
I need to learn to see the calendar again!!!